packages/api-nest/src/queue/services/html-sanitizer.service.ts

```
@@ -0,0 +1,292 @@
+        ALLOWED_ATTR: allowedAttr,
+        ALLOW_DATA_ATTR: options?.allowDataAttributes || false,
+        KEEP_CONTENT: true, // Keep text content even if tag is removed
+        RETURN_DOM: false,
+        RETURN_DOM_FRAGMENT: false,
+        FORCE_BODY: false,
+      }
+
+      const sanitized = String(DOMPurify.sanitize(html, config))
+
+      // Log if significant content was removed (potential security issue)
+      const removedRatio = 1 - sanitized.length / html.length
```

ESLint

[error] Unexpected mix of '-' and '/'. Use parentheses to clarify the intended order of operations.

(no-mixed-operators)